include/constants/battle/condition.h

```diff
@@ -146,24 +148,6 @@
 #define FIELD_CONDITION_GRAVITY_SHIFT       12
 #define FIELD_CONDITION_TRICK_ROOM_SHIFT    16
 
-#define OVERWORLD_WEATHER_CLEAR         0
-#define OVERWORLD_WEATHER_CLOUDY        1
-#define OVERWORLD_WEATHER_RAINING       2
-#define OVERWORLD_WEATHER_HEAVY_RAIN    3
-#define OVERWORLD_WEATHER_THUNDERSTORM  4
-#define OVERWORLD_WEATHER_SNOWING       5
-#define OVERWORLD_WEATHER_HEAVY_SNOW    6
-#define OVERWORLD_WEATHER_BLIZZARD      7
-#define OVERWORLD_WEATHER_CLEAR_8       8
-#define OVERWORLD_WEATHER_SLOW_ASHFALL  9
-#define OVERWORLD_WEATHER_SANDSTORM     10
-#define OVERWORLD_WEATHER_HAILING       11
-#define OVERWORLD_WEATHER_SPIRITS       12
-#define OVERWORLD_WEATHER_CLEAR_13      13
-#define OVERWORLD_WEATHER_FOG           14
-#define OVERWORLD_WEATHER_DEEP_FOG      15
-#define OVERWORLD_WEATHER_DARK_FLASH    16
-
 // these are only for the Battle Frontier
 #define OVERWORLD_WEATHER_HARSH_SUN     1001
 #define OVERWORLD_WEATHER_TRICK_ROOM    1002
```

Comment:
We should move these, as well.

Reply:
Aren't these only used for the battle arcade?